static/app/views/dashboards/widgets/tableWidget/tableWidgetVisualization.tsx

```diff
@@ -0,0 +1,109 @@
+import {useTheme} from '@emotion/react';
+
+import type {GridColumnOrder} from 'sentry/components/gridEditable';
+import GridEditable from 'sentry/components/gridEditable';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import type {
+  TabularColumn,
+  TabularData,
+} from 'sentry/views/dashboards/widgets/common/types';
+import {
+  renderDefaultBodyCell,
+  renderDefaultHeadCell,
+} from 'sentry/views/dashboards/widgets/tableWidget/defaultTableCellRenderers';
+
+interface TableWidgetVisualizationProps {
```

Comment:
Could you add prop documentation here? Check out `TimeSeriesWidgetVisualizationProps` docs, each prop has a whole section to explain what it does. If you do that, you won't have to have as much documentation in the story, and you can focus on explaining table behaviour!